Wikipedia/Code/NotificationsCenterAction.swift

```diff
@@ -30,6 +30,13 @@ enum NotificationsCenterAction: Equatable {
             }
         }
     }
+
+    var actionData: NotificationsCenterActionData? {
+        switch self {
+        case .notificationSubscriptionSettings(let data), .markAsReadOrUnread(let data), .custom(let data):
```

Comment:
Sorry you had to do this - I'm hating this enum that I made more and more as time goes on 😁.